docs/source/user/settings.rst

```diff
@@ -200,6 +200,38 @@ This setting can be used to set the maximum size limit for firmware images, eg:
     The numeric value represents the size of files in bytes.
     Setting this to ``None`` will mean there's no max size.
 
+
+``OPENWISP_RADIUS_CALLED_STATION_ID``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``{}``
+
+This setting should contain information of OpenVPN Management Interface for fetching
+information of client to update ``called_station_id`` of RADIUS sessions using
+`convert_called_station_id <management_commands.html#convert-called-station-id>`_ command.
+
+The dictionary should contain configuration in following format:
+
+.. code-block:: python
+
+    OPENWISP_RADIUS_CALLED_STATION_IDS = {
+        'organization_slug': {
```

Comment:
should `organization_slug` contain the actual organization slug?

Reply:
yes, I have update the example to be more explicit